### ISSUE TITLE
Can't setup ulogger server (Uncaught PDOException)

### ISSUE THREAD

**Issue by masscream** [bfabiszewski/ulogger-server#197]
Trying to run ulogger server on my RHEL machine with MariaDB. However, I can't pass the setup successfully. After the first screen I get the HTTP 500 error.

php log
```
[11-Jul-2022 13:45:48 Europe/Prague] PHP Fatal error:  Uncaught PDOException: There is no active transaction in /var/www/ulogger/scripts/setup.php:93
Stack trace:
#0 /var/www/ulogger/scripts/setup.php(93): PDO->rollBack()
#1 {main}
  thrown in /var/www/ulogger/scripts/setup.php on line 93
```
Not sure what the cause is. The database is running and is accessible, when I browse it, the script has even created the table records... but for some reason it fails then, probably before the user creation.
```

[root@server ulogger]# php --version
PHP 8.0.21 (cli) (built: Jul  6 2022 10:13:53) ( NTS gcc x86_64 )
Copyright (c) The PHP Group
Zend Engine v4.0.21, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.21, Copyright (c), by Zend Technologies
[root@server ulogger]# mysql --version
mysql  Ver 15.1 Distrib 10.3.32-MariaDB, for Linux (x86_64) using readline 5.1
```

**Comment by bfabiszewski** [comment]
Which version of ulogger is that?
Try latest git version, because there have been many changes since last release.

**Comment by bfabiszewski** [comment]
I've just released new [version 1.1](https://github.com/bfabiszewski/ulogger-server/releases/tag/v1.1).
This problem should be fixed in this release.
